src/operators/suffix.ts

```diff
@@ -1,3 +1,4 @@
+import '../typeAugmentations/allow';
```

Comment:
This is a pattern we use in fs.js. Ie11 supports maps, but technically es5 does not. See workaround here.

Reply:
curious: is this to make typescript happy? (`Map` sin't showing any errors for me, but it seems like it should since we only have `es5` libs for ts?)

Reply:
Yeah I was getting an error in my editor for `Map`. (Didn't try using the build system 🤔 ) 

I'll see if we can remove or refactor.